ibis/backends/dask/execution/generic.py

```diff
@@ -136,11 +137,17 @@
     ],
     ops.Distinct: [((dd.DataFrame,), execute_distinct_dataframe)],
 }
+
 register_types_to_dispatcher(execute_node, DASK_DISPATCH_TYPES)
 
 execute_node.register(DaskTable, DaskClient)(execute_database_table_client)
 
 
+@execute_node.register(ops.ValueList, collections.abc.Sequence)
```

Comment:
Why do you need to add this?

Reply:
We need to use this specific function since it uses the dask `execute`

Reply:
Ok